RELEASE/scripts/CONSUME.ash

```diff
@@ -1299,6 +1299,11 @@ void main(string command)
 	{
 		buffer b;
 		b.append_diet(d);
+		string userForbidden = get_property("forbiddenStores");
+		if (!userForbidden.contains_text("3408540")) {
+			set_property("forbiddenStores", userForbidden + ",3408540"); // forbid Dance Police
```

Comment:
Should probably only add the comma if the list isn't empty. It probably won't break anything, but it's kinda ugly (this matters more if we decide to keep the store permanently forbidden).